docs/sdk/src/polkadot_sdk/cumulus.rs

```diff
@@ -42,7 +42,6 @@
 //!
 //! [FRAME]: crate::polkadot_sdk::frame_runtime
 
-#![deny(rustdoc::broken_intra_doc_links)]
```

Comment:
Unclear if this lint must live here, if so it may need to be feature gated as it breaks the minimal docs build